src/site/antora/modules/ROOT/pages/development.adoc

```diff
@@ -87,32 +87,32 @@ Pitch it in a {logging-services-url}/support.html#discussions-maintainer[maintai
 Go ahead and push it.
 
 . Otherwise, submit a pull request.
-Make sure a changelog entry is attached and `./mvnw verify site` succeeds.
+Ensure a changelog entry is attached and `./mvnw verify site` succeeds.
 +
 [TIP]
 ====
-You are strongly advised to spar with another maintainer first (see {logging-services-url}/support.html#discussions-maintainer[maintainer discussion channels]) before starting to code.
+We strongly advise you to spar with another maintainer first (see {logging-services-url}/support.html#discussions-maintainer[maintainer discussion channels]) before starting to code.
```

Comment:
Is "spar" the right term here? I only know that in context of practice martial arts fights (and apparently it's some part of a ship). I might go with something like "discuss" here.